src/main/resources/migrations/00014_alter_table_gocardless_payments_charge_date.sql

```diff
@@ -0,0 +1,5 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:alter_table-gocardless-payments-charge-date
+ALTER TABLE gocardless_payments ADD COLUMN charge_date TIMESTAMP;
```

Comment:
Do we usually use `TIMESTAMP` for things like this?

Reply:
We usually use `TIMESTAMP WITH TIME ZONE DEFAULT (now() AT TIME ZONE 'utc')`, but in this case gocardless' date is of format "2014-05-21", so no timezone there

Reply:
So do we want `DATE` then?

Reply:
That's why I was thinking of using LocalDate in the code to represent it, instead of ZonedDateTime

Reply:
Oh cool, I just discovered that `TIMESTAMP` has resolution of 1 microsecond and `DATE` of 1 day. Considering that GC just gives back the day, I guess `DATE` fits this case better :) will change, thanks

Reply:
Definitely use `LocalDate` in the code: precise types are awesome in all kinds of ways!

Reply:
👍